relays/ethereum/src/ethereum_deploy_contract.rs

```diff
@@ -14,12 +14,19 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges Common.  If not, see <http://www.gnu.org/licenses/>.
 
-use crate::ethereum_client::{self, EthereumConnectionParams, EthereumSigningParams};
-use crate::substrate_client::{self, SubstrateConnectionParams};
+use crate::ethereum_client::{
+	EthereumConnectionParams, EthereumHighLevelRpc, EthereumRpcClient, EthereumSigningParams,
+};
+use crate::rpc::SubstrateRpc;
+use crate::substrate_client::{SubstrateConnectionParams, SubstrateRpcClient};
 use crate::substrate_types::{Hash as SubstrateHash, Header as SubstrateHeader};
+
 use codec::{Decode, Encode};
 use num_traits::Zero;
 
+// to encode/decode contract calls
+ethabi_contract::use_contract!(bridge_contract, "res/substrate-bridge-abi.json");
```

Comment:
It'll generate 2x more code (with incompatible types?). Probably only include once?